### ISSUE TITLE
Chaos Orb Recipe doesn't include ilvl 76 items

### ISSUE THREAD

**Issue by klaus-o-nielsen** [ChaosRecipeEnhancer/ChaosRecipeEnhancer#363]
When running "Chaos Orb Recipe" and "Prevent Low Item Level Gear" checked the filter dosen't include items over ilvl 75

**Comment by HiMarioLopez** [comment]
So basically, filter highlighting does 2 things.

![image](https://user-images.githubusercontent.com/23390842/168527610-00d97e60-3b98-4992-8499-a4616888d3ad.png)

If the "Always Active" checkbox is on, ALL items higher than item level 60 will be shown for that item type.

If you don't have that selected, and are doing the Chaos Recipe, we only highlight items on your filter than are valid to the Chaos Recipe, which is ONLY item level 60 to 74 

![image](https://user-images.githubusercontent.com/23390842/168527788-ab643854-e595-4855-9098-f34956f12679.png)

Source: https://www.poewiki.net/wiki/Vendor_recipe_system

So basically, assuming you don't have the "Always Active" box ticked for the item you want, it's working as intended.
